src/components/KTable/KTable.vue

```diff
@@ -1028,6 +1200,25 @@ export const defaultSorter = (key: string, previousKey: string, sortOrder: strin
       padding: var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
       text-align: left;
 
+      &.resizable {
+        min-width: 20px;
```

Comment:
suggestion: I think the minimum column width should be at least large enough to see the first character of the column name (maybe `40px`?); otherwise, you can see table cell data but not the header and it's a bit odd

![image](https://github.com/Kong/kongponents/assets/2229946/2f5e17ea-8da1-43a4-9b28-6f98212e1b15)